doc/source/api.rst

```diff
@@ -520,6 +520,9 @@ Session
    :toctree: _generated/
 
    Session
+   local_arrays
+   global_arrays
+   load_example_data
```

Comment:
why here?

Reply:
because those functions return sessions/are a way to create a session. Do you see a better place?

Reply:
No, LGTM.